floris/optimization/layout_optimization/layout_optimization_scipy.py

```diff
@@ -18,9 +18,10 @@ def __init__(
         solver='SLSQP',
         optOptions=None,
         enable_geometric_yaw=False,
+        use_value=False,
     ):
         """
-        _summary_
+        _summary_ TODO: write summary
```

Comment:
@ejsimley if you want I'm happy to take a crack at this and commit to your branch.  Also, elsewhere in FLORIS we have the connection that the class docstring comes before the init function, in the same commit I can move this above?

Reply:
Yes, please take care of that if you'd like, thanks. This is how the docstrings were before I started editing, so didn't feel like changing too much, but probably a good time to update it!

Reply:
will do!